### ISSUE TITLE
How to fetch the Prediction Probabilities of the Predicted Tokens 

### ISSUE THREAD

**Issue by ManasiPat** [LUMIA-Group/rasat#4]
We want to get the **prediction probabilities of the predicted tokens** (on the top beam) without constrained decoding (PICARD). How can we get these values? @JiexingQi  Can you please point to the code for the same? Thanks.

**Comment by Monstarrr** [comment]
Without constrained decoding, our decoder is just a normal T5 decoder, which means the probability you want is the lm_logits in line 1804 of t5_relation_model.py after softmax func. If you want to get the exact probability, maybe you should refer to the original transformer code in generation part.
